src/realm.h

```diff
@@ -2833,6 +2833,11 @@ typedef struct realm_user_identity {
     realm_auth_provider_e provider_type;
 } realm_user_identity_t;
 
+typedef void (*realm_return_apikey_func_t)(realm_userdata_t userdata, realm_app_user_apikey_t*,
+                                           const realm_app_error_t*);
+typedef void (*realm_return_apikey_list_func_t)(realm_userdata_t userdata, realm_app_user_apikey_t[], size_t count,
```

Comment:
You need to add an entry in the CHANGELOG specifying what you have done, probably under breaking changes, for example in this case: 
`realm_app_user_apikey_provider_client_fetch_apikeys`  you have changed the signature of this function, thus all the C-API consumers will have to adapt to your new code.